test/e2e/logging_soak.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/test/e2e/framework"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+)
+
+// This test is designed to run and confirm that logs are being generated at a large scale, and that they can be grabbed by the kubelet.
+// By running it repeatedly in the background, you can simulate large collections of chatty containers.
+// This can expose problems in your docker configuration (logging), log searching infrastructure, to tune deployments to match high load
+// scenarios.  TODO jayunit100 add this to the kube CI in a follow on infra patch.
+var _ = framework.KubeDescribe("Logging soak [Feature:LoggingSoak] [Slow] [Disruptive]", func() {
+
+	f := framework.NewDefaultFramework("logging-soak")
+
+	// Not a global constant (irrelevant outside this test), also not a parameter (if you want more logs, use --scale=).
+	kbRateInSeconds := 1 * time.Second
+	totalLogTime := 2 * time.Minute
+
+	It(fmt.Sprintf("should survive logging 1KB every %v seconds, for a duration of %v, scaling up to %v pods per node", kbRateInSeconds, totalLogTime, framework.TestContext.Scale), func() {
```

Comment:
So here the `framework.TestContext.Scale` parameter is interpreted to mean # pods per node.  So this would be the first use of this parameter.

Would there be any harm is just defining its meaning from the outset that it means "# of pods per node"?